api/src/main/java/ru/urururu/sanity/api/cfg/ConstCache.java

```diff
@@ -24,84 +25,11 @@ public RValue getNull(Type type) {
     }
 
     public RValue getFunction(String name, Type type) {
+        name = fixName(name);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "name". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)